desktop/core/src/desktop/api2_tests.py

```diff
@@ -37,6 +37,113 @@ def setUp(self):
+    ''' \\"name\\": \\"wf2\\", \\"hasKillNode\\": true, \\"versions\\": [\\"uri:oozie:workflow:0.4\\", \\"uri:oozie:workflow:'''\
+    '''0.4.5\\", \\"uri:oozie:workflow:0.5\\"], \\"isDirty\\": true, \\"movedNode\\": null, \\"linkMapping\\": {\\"33430f0f-ebfa-'''\
+    '''c3ec-f237-3e77efa03d0a\\": [], \\"3f107997-04cc-8733-60a9-a4bb62cebffc\\": [\\"33430f0f-ebfa-c3ec-f237-3e77efa03d0a\\"], '''\
+    '''\\"17c9c895-5a16-7443-bb81-f34b30b21548\\": []}, \\"nodeIds\\": [\\"3f107997-04cc-8733-60a9-a4bb62cebffc\\", \\"33430f0f-'''\
+    '''ebfa-c3ec-f237-3e77efa03d0a\\", \\"17c9c895-5a16-7443-bb81-f34b30b21548\\"], \\"nodes\\": [{\\"name\\": \\"Start\\", \\"'''\
+    '''associatedDocumentLoading\\": true, \\"children\\": [{\\"to\\": \\"33430f0f-ebfa-c3ec-f237-3e77efa03d0a\\"}], '''\
+    '''\\"actionParametersFetched\\": false, \\"properties\\": {}, \\"associatedDocumentUuid\\": null, \\"type\\": \\"start'''\
+    '''-widget\\", \\"id\\": \\"3f107997-04cc-8733-60a9-a4bb62cebffc\\", \\"actionParameters\\": []}, {\\"name\\": \\"End\\",'''\
+    ''' \\"associatedDocumentLoading\\": true, \\"children\\": [], \\"actionParametersFetched\\": false, \\"properties\\":'''\
+    ''' {\\"body\\": \\"\\", \\"cc\\": \\"\\", \\"to\\": \\"\\", \\"enableMail\\": false, \\"attachment\\": \\"\\", '''\
+    '''\\"content_type\\": \\"text/plain\\", \\"subject\\": \\"\\"}, \\"associatedDocumentUuid\\": null, \\"type\\": '''\
+    '''\\"end-widget\\", \\"id\\": \\"33430f0f-ebfa-c3ec-f237-3e77efa03d0a\\", \\"actionParameters\\": []}, {\\"name\\":'''\
+    ''' \\"Kill\\", \\"associatedDocumentLoading\\": true, \\"children\\": [], \\"actionParametersFetched\\": false, '''\
+    '''\\"properties\\": {\\"body\\": \\"\\", \\"cc\\": \\"\\", \\"to\\": \\"\\", \\"enableMail\\": false, \\"message\\":'''\
+    ''' \\"Action failed, error message[${wf:errorMessage(wf:lastErrorNode())}]\\", \\"subject\\": \\"\\"}, \\"associatedDocu'''\
+    '''mentUuid\\": null, \\"type\\": \\"kill-widget\\", \\"id\\": \\"17c9c895-5a16-7443-bb81-f34b30b21548\\", '''\
+    '''\\"actionParameters\\": []}], \\"id\\": null, \\"nodeNamesMapping\\": {\\"33430f0f-ebfa-c3ec-f237-3e77efa03d0a\\": '''\
+    '''\\"End\\", \\"3f107997-04cc-8733-60a9-a4bb62cebffc\\": \\"Start\\", \\"17c9c895-5a16-7443-bb81-f34b30b21548\\": '''\
+    '''\\"Kill\\"}, \\"uuid\\": \\"92a7dd47-d8c8-b2d8-2895-440ccbc94198\\"}}",\n    "extra": "",\n    "search": null,\n   '''\
+    ''' "last_modified": "2023-01-05T23:08:27.853",\n    "version": 1,\n    "is_history": false,\n    "is_managed": false,\n '''\
+    '''   "is_trashed": false,\n    "parent_directory": [\n      "117db535-78b6-42e5-92bf-dbafcae015a7",\n      1,\n      '''\
+    '''false\n    ],\n    "dependencies": []\n  }\n}\n]\n'''
+    
+    response = client.post("/desktop/api2/doc/import", {'documents': json.dumps(doc)})
+    status = json.loads(response.content)['status']
```

Comment:
Perhaps some assertion on the documents created from the import?

Reply:
Okay, checking where we can add this